<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-KGDMBK1V" failures="0" tests="4" name="com.testng.parametersdemo.InvocationTimeOutDemo" time="4.161" errors="2" timestamp="21 Oct 2021 07:17:16 GMT" skipped="0">
  <testcase classname="com.testng.parametersdemo.InvocationTimeOutDemo" name="m4" time="3.021">
    <error message="Method com.testng.parametersdemo.InvocationTimeOutDemo.m4() didn&#039;t finish within the time-out 3000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testng.parametersdemo.InvocationTimeOutDemo.m4() didn't finish within the time-out 3000
at java.base@15.0.2/java.io.FileOutputStream.writeBytes(Native Method)
at java.base@15.0.2/java.io.FileOutputStream.write(FileOutputStream.java:347)
at java.base@15.0.2/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
at java.base@15.0.2/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
at java.base@15.0.2/java.io.PrintStream.write(PrintStream.java:570)
at java.base@15.0.2/sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:242)
at java.base@15.0.2/sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:321)
at java.base@15.0.2/sun.nio.cs.StreamEncoder.flushBuffer(StreamEncoder.java:110)
at java.base@15.0.2/java.io.OutputStreamWriter.flushBuffer(OutputStreamWriter.java:181)
at java.base@15.0.2/java.io.PrintStream.writeln(PrintStream.java:723)
at java.base@15.0.2/java.io.PrintStream.println(PrintStream.java:1028)
at app//com.testng.parametersdemo.InvocationTimeOutDemo.m4(InvocationTimeOutDemo.java:26)
at java.base@15.0.2/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@15.0.2/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base@15.0.2/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@15.0.2/java.lang.reflect.Method.invoke(Method.java:564)
at app//org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at app//org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at app//org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:40)
at java.base@15.0.2/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base@15.0.2/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@15.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base@15.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base@15.0.2/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- m4 -->
  <testcase classname="com.testng.parametersdemo.InvocationTimeOutDemo" name="m3" time="1.126">
    <error message="Method com.testng.parametersdemo.InvocationTimeOutDemo.m3() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testng.parametersdemo.InvocationTimeOutDemo.m3() didn't finish within the time-out 1000
]]>
    </error>
  </testcase> <!-- m3 -->
  <testcase classname="com.testng.parametersdemo.InvocationTimeOutDemo" name="m2" time="0.003"/>
  <testcase classname="com.testng.parametersdemo.InvocationTimeOutDemo" name="m1" time="0.011"/>
</testsuite> <!-- com.testng.parametersdemo.InvocationTimeOutDemo -->
